userwidgets/tiles.sessionTile/modules/require/sessionTileController.js

```diff
@@ -49,6 +65,23 @@ define(function() {
             hours = hours < 10 ? "0" + hours : hours;
             var minutes = date.getMinutes() < 10 ? "0" + date.getMinutes() : date.getMinutes();
             return hours + ":" + minutes + ":" + " " + am_pm;
+        },
+       /**
+         *	@function sessionToMySchedule
+         * 	@description This function is used to add sessions to strore in phone
+         * 	@private
+         */
+      sessionToMySchedule:function(){
+        this.isAddedToMySchedule=true;
+        this.sessionData.isAddedToMySchedule=true;
+        this.view.imgStatus.src=this.myScheduleIndicatorImage;
+        this.view.addAgendaContainer.skin=this.agendaContainerSkin;
+        var myAgendaData=kony.store.getItem("myAgendaData");
+        if(kony.sdk.isNullOrUndefined(myAgendaData)){
+          myAgendaData=[];
         }
+        myAgendaData.push(this.sessionData);
```

Comment:
store session id map instead of whole array

Reply:
Changed Array to Dict